### ISSUE TITLE
The clipboard from the remote to the local system is limited to 511 symbols

### ISSUE THREAD

**Issue by Rush-iam** [Xpra-org/xpra-html5#241]
It works fine for any size from the local system to the remote, but in the opposite way, it is possible only up to 511 symbols.
The client doesn't update clipboard content when copying a text block over 511 symbols.

- client version - latest commit here
- OS - ChromeOS 112
- Browser - Chrome 112
- command: `xpra start --bind-ws=0.0.0.0:9999`

**Comment by totaam** [comment]
What OS?
Is this Ubuntu again?
If so, there is a bug with brotli: #3824.

**Comment by Rush-iam** [comment]
Yes, server runs on Ubuntu 22.04

**Comment by Rush-iam** [comment]
Oh, I haven't brotli installed, but after `apt install brotli` now I see the errors in the log the same as https://github.com/Xpra-org/xpra/issues/3824

Unfortunately, I can't downgrade brotli version - there is only one available - 1.0.9-2build6.
Here is the changelog of brotli apt package for Ubuntu that may lead you to a reason:
https://changelogs.ubuntu.com/changelogs/pool/main/b/brotli/brotli_1.0.9-2build6/changelog

```
brotli (1.0.9-2build6) jammy; urgency=medium
  * No-change rebuild for ppc64el baseline bump.

brotli (1.0.9-2build5) jammy; urgency=medium
  * No-change rebuild with Python 3.10 only

brotli (1.0.9-2build4) jammy; urgency=medium
  * No-change rebuild to add python3.10.

brotli (1.0.9-2build3) impish; urgency=medium
  * No-change rebuild to build packages with zstd compression.

brotli (1.0.9-2build2) hirsute; urgency=medium
  * No-change rebuild to drop python3.8 extensions.

**Comment by Rush-iam** [comment]
I tried building brotli from [sources](https://github.com/google/brotli), but it has the same error. Seems like not related to brotli binaries itself

**Comment by totaam** [comment]
I'm reasonably confident that this has been fixed in #3824, you can find a workaround in https://github.com/Xpra-org/xpra/issues/3824#issuecomment-1513907124
